front/src/modules/rollingStock/components/RollingStockEditor/CurveSpreadsheet.tsx

```diff
@@ -77,30 +80,69 @@ const CurveSpreadsheet = ({
         ...(isDefaultCurve ? { default_curve: formattedCurve } : {}),
       },
     };
+
+    // Set the updated effort curves
```

Comment:
Not necessary, since the code is quite straightforward
```suggestion
```

Reply:
Done